yarn.lock

```diff
@@ -9577,7 +9585,7 @@ triple-beam@^1.2.0, triple-beam@^1.3.0:
   version "1.3.0"
   resolved "https://registry.yarnpkg.com/triple-beam/-/triple-beam-1.3.0.tgz#a595214c7298db8339eeeee083e4d10bd8cb8dd9"
 
-ts-md5@^1.2.2:
+ts-md5@*, ts-md5@^1.2.2:
```

Comment:
I don't think `*` as a version is good, what would it pull for end products if `1.3` is released? `1.2.4` and `1.3`?